common/comms/proton_queue_adaptor.py

```diff
@@ -35,25 +37,28 @@ def __init__(self, **kwargs) -> None:
         self.host = kwargs.get('host')
         logger.info('000', 'Initialized proton queue adaptor for {host}', {'host': self.host})
 
-    async def send_async(self, message: str) -> None:
+    async def send_async(self, message: str, properties: Dict[str, Any] = None) -> None:
         logger.info('008', 'Sending message asynchronously.')
+        payload = self.__construct_message(message, properties=properties)
         await tornado.ioloop.IOLoop.current() \
-            .run_in_executor(executor=None, func=lambda: self.__send(self.__construct_message(message)))
+            .run_in_executor(executor=None, func=lambda: self.__send(payload))
 
-    def send_sync(self, message: str) -> None:
+    def send_sync(self, message: str, properties: Dict[str, Any] = None) -> None:
         logger.info('009', 'Sending message synchronously.')
-        self.__send(self.__construct_message(message))
+        self.__send(self.__construct_message(message, properties=properties))
```

Comment:
Optional: It would be more consistent to extract the `__construct_message` call to a `payload` variable as you've done in `send_async`.